### PR TITLE
Improved get metrics to account for Key not found errors

### PR DESCRIPTION
**Pull request by guscarreon**

### CODE REVIEW

**Review by VeronikaSolovei9 — COMMENTED** [review]
LGTM, but it needs another eye on it.

**Review by VeronikaSolovei9 — DISMISSED** [review]

File: metrics/prometheus/prometheus_test.go [hunk 1]
```diff
@@ -362,7 +394,7 @@ func TestMetricCountGatekeeping(t *testing.T) {
 	}
 
 	// This assertion provides a warning for newly added high-cardinality non-adapter specific metrics. The hardcoded limit
-	// is an arbitrary soft ceiling. Thought should be given as to the value of the new metrics if you find yourself
-	// needing to increase this number.
+	//   is an arbitrary soft ceiling. Thought should be given as to the value of the new metrics if you find yourself
```
Comment (mansinahar) [inline metrics/prometheus/prometheus_test.go:397]: Nitpick: What's the reason behind adding additional spaces here? It's standard to have a single space at the start in comments

File: utils/errors.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+// Missing UUID error
+type MissingKeyError struct{}
+
+func (e MissingKeyError) Error() string {
+	return "missing required parameter uuid"
+}
+
+// Invalid UUID lenght
```
Comment (mansinahar) [inline utils/errors.go:23]: Typo: `length`

File: metrics/prometheus/prometheus_test.go (hunk 1 above)
Comment (guscarreon) [reply metrics/prometheus/prometheus_test.go:397]: typo, changed

File: utils/errors.go (hunk 2 above)
Comment (guscarreon) [reply utils/errors.go:23]: I always misspell this word. Changed

File: endpoints/get.go [hunk 3]
```diff
@@ -80,13 +81,13 @@ func handleException(w http.ResponseWriter, err error, status int, uuid string)
...
 
-	logError(err, msg)
+	toLogger(err, msg)
 
 	http.Error(w, msg, status)
 }
 
-func logError(err error, msg string) {
-	if _, isKeyNotFound := err.(backends.KeyNotFoundError); isKeyNotFound {
+func toLogger(err error, msg string) {
```
[the remark this answers is not in the document]
Comment (guscarreon) [reply endpoints/get.go:89]: rolledback

**Review by VeronikaSolovei9 — APPROVED** [review]

**Review by mansinahar — APPROVED** [review]

**Pull request merged**